lighthouse-core/test/lib/i18n/swap-locale-test.js

```diff
@@ -52,6 +52,7 @@ describe('swap-locale', () => {
         redirects: {
           id: 'redirects',
           title: 'Avoid multiple page redirects',
+          doesntExist: 'A string that does not have localized versions',
```

Comment:
should we add a different path to something that doesn't exist then? feels like this is a duplicate test of fakeaudit title otherwise